rpa-bot/1.3/modules/ROOT/pages/configuration.adoc

```diff
@@ -16,29 +16,35 @@ After saving your changes, restart the RPA Bot Service for the changes to take e
 [[common-settings]]
 == Common Settings
 
-image:bot-configurator-common.png[The RPA Bot configurator application showing the Common settings panel, 50%, 50%]
+image:bot-configurator-common.png[The RPA Bot Configurator application showing the Common settings panel, 50%, 50%]
 
 * *RPA Bot ID*
 +
 ID generated during the installation process of RPA Bot, which identifies the bot in RPA Manager (even if the IP or MAC-Address changes). If you need a new ID, uninstall and then reinstall RPA Bot.
 * *RPA Manager to connect*
 +
 The URL address of the RPA Manager instance the bot connects to. There is no default setting.
-* *Use HTTPS as protocol layer*
-+
-Specifies whether the bot connects with RPA Manager via HTTP or HTTPS.
 * *API Key*
 +
 The API key entered during the installation of the robot. You can change this setting.
+* *TEST CONNECTION*
++
+Click *TEST CONNECTION* to verify that the bot can establish a connection to RPA Manager.
 * *Alive interval*
 +
 Time interval in milliseconds after which the bot connects to the RPA Manager to show that it is still alive.
 * *Check interval*
 +
 Time interval in milliseconds after which the bot connects to RPA Manager and checks for provided packages to collect (such as scheduling tasks or activities).
-* *Cleanup workflows on start*
+* *Cleanup activities on start*
++
+Select this option to delete the activities that are no longer assigned to a configuration but are still stored locally in the bot.
+* *Cleanup runtime packages on next start*
 +
-Select this option to delete the workflows that are no longer assigned to a configuration but are still stored locally in the bot.
+Select this option to delete all runtime packages that are stored locally in the bot. The process activities retrieve the required runtime packages from the RPA Manager on the next run. After the runtime packages are deleted, the Bot Configurator deselects this setting.
+* *Maximum number of sessions*
++
+Set the maximum number of Secure Sessions that the bot can use. Enabling this setting, limits but not overwrites the same setting in RPA Manager Bot Management.
```

Comment:
```suggestion
Set the maximum number of secure sessions that the bot can use. Enabling this setting limits, but doesn't overwrite, the same setting in RPA Manager Bot Management.
```

Reply:
What does "limits... the same setting" mean? Does it limit the number of secure sessions but doesn't overwrite the setting elsewhere?

Reply:
Yes
Example
Bot Setting + Manager Setting = Valid Setting
42 + 1 = 1
1 + 42 = 1

Reply:
Can you revise to make clear?

Enabling this setting limits the number of secure sessions, but doesn't overwrite the same setting in RPA Manager Bot Management.